js/windowMgr.js

```diff
@@ -320,7 +320,7 @@ function doCreateMainWindow(initialUrl, initialBounds) {
 
         // only allow window.open to succeed is if coming from same hsot,
         // otherwise open in default browser.
-        if (disposition === 'new-window' && newWinHost === mainWinHost) {
+        if (disposition === 'new-window' && ((newWinHost === mainWinHost) || newWinUrl === 'about:blank')) {
```

Comment:
I would rather this is stored in a variable and that checked here instead of hardcoding it.

Reply:
Done